robust_intelligence_ai_firewall/assets/dashboards/robust_intelligence_ai_firewall_overview.json

```diff
@@ -0,0 +1,591 @@
+{
+  "title": "Robust Intelligence AI Firewall Results",
+  "description": "[[suggested_dashboards]]",
```

Comment:
The dashboard description should include the value of the dashboard

Reply:
Thanks, added a description